tutorials/09-variational-inference/09_variational-inference.jmd

```diff
@@ -154,27 +155,23 @@ For that we need samples:
 
 ```julia
 samples = rand(q, 10000);
-```
-
-```julia
-# setup for plotting
-using LaTeXStrings, StatsPlots
+size(samples)
 ```
 
 ```julia
 p1 = histogram(samples[1, :]; bins=100, normed=true, alpha=0.2, color=:blue, label="")
-density!(samples[1, :]; label="s (ADVI)", color=:blue, linewidth=2)
-density!(samples_nuts, :s; label="s (NUTS)", color=:green, linewidth=2)
-vline!([var(x)]; label="s (data)", color=:black)
-vline!([mean(samples[1, :])]; color=:blue, label="")
+density!(samples[1, :]; ylabel="Density", label="s (ADVI)", color=:blue, linewidth=2)
+density!(samples_nuts, :s; ylabel="Density", label="s (NUTS)", color=:green, linewidth=2)
+vline!([var(x)]; ylabel="Density", label="s (data)", color=:black)
+vline!([mean(samples[1, :])]; ylabel="Density", color=:blue, label="")
```

Comment:
You don't have to set `ylabel` in every plotting call. It should be sufficient to use
```suggestion
p1 = histogram(samples[1, :]; bins=100, normed=true, alpha=0.2, color=:blue, label="", ylabel="density")
density!(samples[1, :]; label="s (ADVI)", color=:blue, linewidth=2)
density!(samples_nuts, :s; label="s (NUTS)", color=:green, linewidth=2)
vline!([var(x)]; label="s (data)", color=:black)
vline!([mean(samples[1, :])]; color=:blue, label="")
```

Reply:
True - just modified it.